app/components/story/StoryCard.js

```diff
@@ -4,18 +4,48 @@ import {
   Text,
   StyleSheet,
   Dimensions,
+  Animated,
 } from 'react-native';
 
 class StoryCard extends Component {
+  constructor(props) {
+    super(props); 
+    this.state = {
+      height: new Animated.Value() 
+    };
+  }
+
+  componentWillMount() {
```

Comment:
is the initial prop ever going to be different?
the logic branch might be unnecessary